### ISSUE TITLE
Mass Obsoletion: focus on branch 'digestive system disorder' (MONDO:0004335)

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#6743]
branch: 'digestive system disorder' (MONDO:0004335)

obsoletion candidate ID | obsoletion candidate label
-- | --
http://purl.obolibrary.org/obo/MONDO_0018532 | adenocarcinoma of liver and intrahepatic biliary tract
http://purl.obolibrary.org/obo/MONDO_0019800 | chronic hepatic porphyria
http://purl.obolibrary.org/obo/MONDO_0015182 | congenital enteropathy involving intestinal mucosa development
http://purl.obolibrary.org/obo/MONDO_0015178 | congenital intestinal transport defect
http://purl.obolibrary.org/obo/MONDO_0017709 | disorder of lipid absorption and transport
http://purl.obolibrary.org/obo/MONDO_0019619 | duplication of the esophagus
http://purl.obolibrary.org/obo/MONDO_0019513 | esophageal malformation
http://purl.obolibrary.org/obo/MONDO_0028618 | gastroenteric neuroendocrine neoplasm
http://purl.obolibrary.org/obo/MONDO_0015111 | gastroesophageal disease
http://purl.obolibrary.org/obo/MONDO_0015509 | hereditary biliary tract disease
http://purl.obolibrary.org/obo/MONDO_0015617 | hereditary gastro-esophageal disease
http://purl.obolibrary.org/obo/MONDO_0018188 | hereditary intestinal polyposis
http://purl.obolibrary.org/obo/MONDO_0015508 | hereditary parenchymatous liver disease
http://purl.obolibrary.org/obo/MONDO_0015180 | intestinal disease due to fat malabsorption
http://purl.obolibrary.org/obo/MONDO_0015179 | intestinal disease due to vitamin absorption anomaly
http://purl.obolibrary.org/obo/MONDO_0019126 | intractable diarrhea of infancy
http://purl.obolibrary.org/obo/MONDO_0015207 | non-syndromic esophageal malformation
http://purl.obolibrary.org/obo/MONDO_0018241 | primary short bowel syndrome
http://purl.obolibrary.org/obo/MONDO_0034661 | syndromic biliary atresia
http://purl.obolibrary.org/obo/MONDO_0015208 | syndromic esophageal malformation

**Comment by twhetzel** [comment]
Can we check why `MONDO:0015183`, 'short bowel syndrome' is not listed as an obsoletion candidate or is it listed in another branch review file?

**Comment by twhetzel** [comment]
@sabrinatoro @nicolevasilevsky - this is ready for review. In addition to the question in the comment block above, I also have a question whether this term name is correct - MONDO:0018009 "non-hypoproteinemic hypertrophic gastropathy". More comments in the review file.

**Comment by nicolevasilevsky** [comment]
@twhetzel I'll review this. Thank you!

**Comment by nicolevasilevsky** [comment]
@twhetzel @sabrinatoro please review my comments on the [spreadsheet](https://docs.google.com/spreadsheets/d/1_WDYrDTna9CCrEi7MMgLNVLK4EqupGLwJHrszxb1oJM/edit#gid=277655657)

**Comment by sabrinatoro** [comment]
Expert review feedback: https://docs.google.com/spreadsheets/d/1JS9qGM5sWVRNGrDLymbegtUnrCLVGNwl/edit#gid=1789041275
TO DO:
- [ ] add new parents ("ROBOT Reviewer-Curator Agree - Stay" Tab)
- [ ] add new comments ("ROBOT Reviewer-Curator Agree - Leave" Tab)
- [ ] add new parents ("ROBOT Reviewer-Stay Curator-Leave" Tab)
- [ ] review "Reviewer-Leave Curator-Stay" Tab [TO DO WILL BE UPDATED ACCORDINGLY]
Note: Once the to-do list is done, 12 terms for which reviewers were unsure will remain to be assessed

**Comment by sabrinatoro** [comment]
There are some comments from the community that should be reviewed, but this is future work. This issue can be closed for now.
